src/referencePackages/src/system.buffers/4.5.1/ref/netstandard1.1/System.Buffers.cs

```diff
@@ -4,7 +4,6 @@
 // ------------------------------------------------------------------------------
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
-[assembly: System.Reflection.AssemblyMetadata("source", "source-build-reference-packages")]
```

Comment:
I just realized that you must be running with an older toolset locally.  You should clean your repo to delete you local artifacts directory and .dotnet.  These attributes were added some time ago now.

Reply:
I just discovered there is a difference in the generated code between running the generator tools and windows vs linux.  This is not good.  I will log and issue and fixup this PR.

Reply:
It appears that there is a difference between the generated code on Windows versus Linux.  I will open an issue on this.  I regen'd everything on linux which reverted all of the updated files.

Reply:
thank you. please link the issue, or ping me one you have it please. I don't see issues on this repo. I will use wsl until that is fixed when I need to make more changes. We plan to update all packages on vstest to latest, so I will have to add few here.

Reply:
https://github.com/dotnet/source-build/issues/4858